packages/mds-policy-author/api.ts

```diff
@@ -108,49 +106,7 @@ const featureCollectionSchema = Joi.object()
   .unknown(true) // TODO
 
 function api(app: express.Express): express.Express {
-  /**
```

Comment:
I guess you determined this `provider_id` check/filter is no longer necessary?

Reply:
I thought we had this hack in just so Blue could log in, in the pre-auth0 world, but after discussing with @avatarneil , I'll put back in the `provider_id` checks to verify that the UUID is present and valid, but not the hack at the end.

Reply:
I don't know what the `hack` is that you are referring to, but my question should probably have been _"Does the access token need to have a valid provider id to access this API?"_ 

I think some applications may proxy requests using a M2M token containing a provider_id claim, but if other applications are doing user-level authentication then there is no mechanism for placing a provider_id in the token presently.

Reply:
@avatarneil actually thinks it's good to rip it all out.

Reply:
Yeah, I agree it's all good to rip out. We definitely don't want a `provider_id` claim to be required for `policy-author` moving forward.